cmd/brew-bundle

```diff
@@ -1,5 +1,17 @@
-#!/usr/bin/env ruby
```

Comment:
`brew-*.rb` will be run by Homebrew's ruby interpreter, i.e. system ruby. While `brew-*` will be run as normal script. In this case, it's the user's ruby interpreter.